### ISSUE TITLE
Route maps on website for GLTC route 3A are not accurate, and multiple incorrect route maps are live on the website

### ISSUE THREAD

**Issue by partytax** [Lynchburg-Urban-Network/lynchburg-road-issues#179]
Non-geographic

Document found by visiting gltconline.com and going to `RIDE GLTC` -> `FIXED ROUTE MAPS TIMETABLE`:
http://gltconline.com/wp-content/uploads/2023/10/RIDERGUIDE_oct_2023_TIMETABLES21484.pdf **NOT ACCURATE FOR 3A**

Some 2022 page found with search engine query "gltc 3a": https://gltconline.com/3a-3b-change-2022/ **ACCURATE FOR 3A**

Some other document found with search engine query "gltc 3a":  https://gltconline.com/route-3a-2/

Basically the correct information should be easy to find on the website, and all incorrect information and past versions of routes should be removed from the website so search engines don't turn them up.

**Comment by partytax** [comment]
Have a personal task to call GLTC customer service during business hours to ensure my 3A ride a few Saturdays ago was typical and that map on website is truly inaccurate.

**Comment by partytax** [comment]
Called customer service a bit ago and they said they'd check it out. Emailing Wendell Watts may be better bet.

**Comment by partytax** [comment]
Discovered that underlying data is wrong. Reported to Wendell Watts:

> Good morning! I found a couple of discrepancies in the GTFS data that I think should be addressed:
> 
> * There is a signed stop on Rivermont Avenue at Marshall Street (3B inbound) that is not in the GTFS data at all. The coordinates 37.43164, -79.15556 would suffice if you don't want to go out and survey.
> * The following stops are associated with route 3A outbound, but should *not* be associated with 3A at all, as 3A goes on Bedford Avenue both ways and not on Rivermont avenue here:
>     * 786315
>     * 786310
>     * 786318
>     * 786319
>     * 786311
>     * 786320
>     * 786308
>     * 786314
>     * 786329
>     * 786317
> 
> Thank you for your time.
> 
> Nathan

**Comment by partytax** [comment]
Received response from Wendell Watts:

> 
>  Hi Nathan,
> 
> The 3A actually does continue straight down Rivermont then makes a Left on the lower end of Bedford Ave. on the first trip of the day. Perhaps that's why you are seeing those stops associated with the 3A.
> 
> Thanks!

**Comment by partytax** [comment]
Was very confused, so called Wendell Watts II and talked to him about this. I _think_ the following happens on the first 3A trip on Monday-Friday and Sunday:

![Screenshot 2024-06-03 at 12-45-53 OpenStreetMap](https://github.com/Lynchburg-Urban-Network/lynchburg-road-issues/assets/29897671/ecc28705-ae96-463d-98af-4b128d982ba4)

**Comment by partytax** [comment]
Map with weird Rivermont route for 3A is actually correct. Latest map available on the website is correct. Need to check those linked maps when I'm on faster internet.

**Comment by partytax** [comment]
Decided that having old, orphaned PDFs on website isn't a big deal. Closing.
